.github/workflows/pr_checks.yml

```diff
@@ -6,13 +6,22 @@ on:
   pull_request:
     branches:
       - '*'
+defaults:
+  run:
+    shell: bash
 jobs:
-  build_and_tests_on_python:
+  build_and_test:
     strategy:
       fail-fast: false
       matrix:
+        os: ['ubuntu-20.04', 'windows-2016']
         python-version: ['3.6', '3.7', '3.8', '3.9']
-    runs-on: ubuntu-latest
+        exclude:
+        # pycocotools-windows doesn't have a wheel for Python 3.9
```

Comment:
Why do we need a wheel? We should be able to build from sources with Cython. 3.9 looks like a thing we need to support (for example, ubuntu 21.04 already uses it).